packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -47,20 +53,34 @@ class DevToolsApp extends StatefulWidget {
 // TODO(https://github.com/flutter/devtools/issues/1146): Introduce tests that
 // navigate the full app.
 class DevToolsAppState extends State<DevToolsApp> {
+  DevToolsAppState([this.conditionalScreens]);
```

Comment:
state objects shouldn't have constructor parameters and should instead read this data off the associated widget.